Source/Tools/LinuxEmulation/VDSO_Emulation.cpp

```diff
@@ -98,6 +100,39 @@ namespace x64 {
       int Result = FHU::Syscalls::getcpu(args->cpu, args->node);
       args->rv = SyscallRet(Result);
     }
+
+    static void getrandom(void* ArgsRV) {
+      struct vgetrandom_opaque_params {
+        uint32_t size_of_opaque_state;
+        uint32_t mmap_prot;
+        uint32_t mmap_flags;
+        uint32_t reserved[13];
+      };
+      static_assert(sizeof(vgetrandom_opaque_params) == sizeof(uint32_t[16]), "Oops");
```

Comment:
`static_assert` can be used without assertion messages, avoiding the need for goofy placeholders.